brand/migrations/0046_alter_commentary_feature_json.py_old

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 5.0.6 on 2024-09-27 20:30
```

Comment:
I'm curious about what's going on with these migrations files here. Since the model didn't change then I imagine they should have been thrown out.

Reply:
Let me double check this.

Reply:
Now I got that, it was my mistake. When I merged main into my branch which had 'black check and isort check PR', it did black my renamed files and I pushed changed without noticing it. I am sorry, I will delete those files.

Reply:
that happens. Especially when someone merges a PR that affects a lot of files. No worries. That's what reviews are for